libs/model/src/aggregates/community/GetMemberships.query.ts

```diff
@@ -0,0 +1,70 @@
+import { Query } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { Op } from 'sequelize';
+import { models } from '../../database';
+import { mustExist } from '../../middleware/guards';
+
+export function GetMemberships(): Query<typeof schemas.GetMemberships> {
+  return {
+    ...schemas.GetMemberships,
+    auth: [],
+    secure: false,
+    body: async ({ payload }) => {
+      const { community_id, address, topic_id } = payload;
+
+      const addr = await models.Address.findOne({
+        where: { address, community_id },
+      });
+      mustExist('Address', addr);
+
+      const groups = await models.Group.findAll({
+        where: { community_id },
+        include: [
+          {
+            model: models.GroupPermission,
+            attributes: ['topic_id', 'allowed_actions'],
+            where: topic_id ? { topic_id } : undefined,
+          },
+        ],
+      });
+
+      // TODO: shouldn't refresh (mutate) in a query... this is eventually consistent
+      // const memberships = await refreshMembershipsForAddress(
```

Comment:
This fundamentally changes the functionality compared to the old route, no? How are we refreshing the membership now if it has expired? Maybe this should be a command and not a query...

Reply:
I don't know enough about this system to provide good answers. I don't know what kind of consistency the client expects but I would imagine that we should invalidate a membership as soon as possible after it expires. As such, I think the `refreshMembershipsForAddress` pattern is good for our purposes. It handles caching (i.e. debouncing) and refresh and will never return an inconsistent value (i.e. membership expired but considered fresh). I don't think it is worth the effort at this time to try to modify the flow to fit in F&F or scheduled refreshing. Again, if side effects are an issue then we should just make this a command and call it a day. No need to overcomplicate it atm.

Reply:
We already have a command for this, not need to duplicate the code in another file. 
Let's wait for @ryanbennettvoid to provide his input since he's probably more familiar with this functionality.